scripts/auto-rebase/last_rebase.sh

```diff
@@ -1,2 +1,3 @@
 #!/bin/bash -x
+# shellcheck disable=all
```

Comment:
This file is automatically generated/updated when rebasing.
If there are warnings here, they need to be fixed in rebase.sh script.

Reply:
Do we get any errors for this file? If not, then we don't have to add that line

Reply:
Oh there are no warnings on that file, I added as a preemptive step, if that file is generated we can remove it

Reply:
Should we add a comment to it for the future?

Something like
```
# DO NOT EDIT
# Auto Generated File
```

Reply:
Yes, this should be done in the rebase.sh which generates this file.

Reply:
Sure, I'll create another PR for these other changes, and keep this one just styling